frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -164,7 +164,11 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
           )}
           {catalogModelDetailsUrl && (
             <DashboardDescriptionListGroup title="Registered from" isEmpty={!mv.id}>
-              <Link to={catalogModelDetailsUrl} data-testid="registered-from-catalog">
+              <Link
+                to={catalogModelDetailsUrl}
+                data-testid="registered-from-catalog"
+                style={{ color: 'var(--pf-t--global--link--Color)', textDecoration: 'none' }}
+              >
```

Comment:
This is the wrong area to change, this link is for versions registered from the model catalog which don't have this bug (they have no project icon). The change needs to actually be made in the ModelVersionPipelineDescription file where we render `<img style={{ height: 24 }} src={typedObjectImage(ProjectObjectType.project)} alt="" />`, that image needs to invert in dark mode (i don't think CSS color vars will work on it but not sure).

Reply:
Please also restore this change to how it was before this PR.